scripts/archive-build-logs-daily.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console */
+const moment = require('moment');
+const PromisePool = require('@supercharge/promise-pool');
+const { Op } = require('sequelize');
+const { Build } = require('../api/models');
+const BuildLogs = require('../api/services/build-logs');
+
+/**
+ * Archive build logs for all builds that completed on the provided date
+ *
+ * @param {string} dateStr - 'YYYY-MM-dd'
+ */
+async function runArchiveBuildLogsDaily(dateStr) {
+  try {
+    const startDate = moment(dateStr);
+    const endDate = startDate.clone().add(1, 'days');
+
+    console.log(`\nQuerying for all builds completed between ${startDate} and ${endDate}.`);
+
+    const builds = await Build.findAll({
+      attributes: ['id'],
+      where: {
+        completedAt: {
+          [Op.gte]: startDate.toDate(),
+          [Op.lt]: endDate.toDate(),
+        },
+      },
+    });
+
+    console.log(`\nFound ${builds.length} builds.`);
+
+    const { errors } = await PromisePool
+      .withConcurrency(5)
```

Comment:
Maybe we can handle more concurrent requests?

Reply:
Once it's up and running, we should have a better idea on tuning concurrency.